docs/kytConfig.md

```diff
@@ -37,6 +37,8 @@ modifyWebpackConfig: (baseConfig, options) => {
 }
 ```
 
+> **NOTE:** The default Webpack configs include [babel-polyfill](https://babeljs.io/docs/usage/polyfill/). If you use `modifyWebpackConfig()` to override entry properties and wish to keep using the polyfill be sure to include it manually.
```

Comment:
Can we update the wording here to mention that the polyfill is included in `entry.main[]` by default and things can be added to the array but should go _after_ the polyfill.